azerothcore/azerothcore-wotlk
# fix(Core/RuinsOfAhnQiraj): Buru

## Changes Proposed:
-  Move scripts to the newer system and modernize the code.
-  Use the proper spell to do damage to players and buru when you kill an egg. [sniffed]
-  Only damage Buru if it is near of the egg destroyed.
-  Implemented its resistance until next phase.
-  All eggs should respawn by default, this will avoid issues like eggs not being spawned after a server restart after a wipe.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/12175
- Updates https://github.com/chromiecraft/chromiecraft/issues/3758

## SOURCE:


## Tests Performed:

- Tested in-game.

## How to Test the Changes:


1. Go to Buru and engage him.
2. Check eggs, phase transition, etc...

## Known Issues and TODO List:


- [ ] Phase transition
